spec/support/shared_examples/requests/post_form.rb

```diff
@@ -20,6 +21,21 @@
         post post_request_path, empty_form_request_body
         expect(response.code).to eq("200")
       end
+
+      it "includes validation errors for the form data" do
+        invalid_form_data.each do |invalid_data|
+          post post_request_path, form_factory => invalid_data.merge(token: correct_form.token)
+
+          invalid_form = build(form_factory, **invalid_data)
+          invalid_form.validate
+
+          invalid_form.errors.messages.values.flatten.each do |error_message|
+            # We include error messages twice, but rspec have no built-in an include twice method yet.
```

Comment:
```suggestion
            # We include error messages twice, but RSpec has no built-in "include twice" method yet.
```

Spotted a typo and then thought this might be a bit clearer.